### DISCUSSION TITLE
Interest in applications

### DISCUSSION THREAD

**Discussion by dtreich** [spstreets/gtfs_manager#59] [General]
Hello! I'm a researcher with [ITDP](itdp.org). I'm excited to see this app and I'm interested in trying it out, maybe sharing it with city DOTs. Is there a live instance I can play with?

Also, please get in touch at taylor.reich@itdp.org to talk about possible collaboration. Thanks. 

Take care.

**Comment by maxwell8888** [comment]
Hi, if by live version you mean a web version, not at the moment, but we plan to at some point. You can subscribe to this issue if you like: https://github.com/spstreets/gtfs_manager/issues/3. Currently you will need to have Rust installed and compile it yourself if you want to run the app. This isn't as hard as it sounds as compiling Rust is *much* easier than languages like C, but still requires you to know your way around a terminal. Honestly though the app is still very rough and simply sharing the video in the README might be best until the app is developed further. Certainly interested in collaboration, I'll drop you an email.
